include/ofi_util.h

```diff
@@ -609,6 +609,7 @@ struct util_wait_fd {
 	fi_epoll_t		epoll_fd;
 	struct dlist_entry	fd_list;
 	fastlock_t		lock;
+	ofi_atomic32_t		ref;
```

Comment:
Good catch, but this doesn't look like the correct fix.  We need the reference counting on the fd_entry.

Reply:
Thanks! Nice point